cmake/PlasmaTargets.cmake

```diff
@@ -1,26 +1,32 @@
 include(BuildOptimizations)
+include(QtDeployment)
 #TODO (#854): include(Sanitizers)
 
 function(plasma_executable TARGET)
     cmake_parse_arguments(_pex
-        "WIN32;QT_GUI;EXCLUDE_FROM_ALL"     # Flags
-        ""                                  # Single values
-        "SOURCES"                           # Multiple values
+        "WIN32;CLI_TOOL;QT_TOOL;CLIENT;EXCLUDE_FROM_ALL"

```

Comment:
FWIW, I am planning to convert plCrashHandler, a client app, into a Qt GUI to allow crash submissions. I think this function would be better both with that in mind and in general by retaining `QT_GUI`, and having just `TOOL` and `CLIENT` as your install switches. Whether to install into tools_cli or tools_gui could be a switch on `if(_pex_WIN32 OR _pex_QT_GUI)`